pkg/controller/common/util.go

```diff
@@ -1494,16 +1494,14 @@ func NewImportDataVolume(name string) *cdiv1.DataVolume {
 func GetCloneSourceInfo(dv *cdiv1.DataVolume) (sourceType, sourceName, sourceNamespace string) {
```

Comment:
This function still has unused named return parameters

Reply:
I kept these on purpose. The returned types are all strings, so I think the named return types serve as documentation.